crates/ethereum/reth/Cargo.toml

```diff
@@ -102,6 +105,7 @@ node = [
     "trie",
 ]
 pool = ["dep:reth-transaction-pool"]
+eth-wire = ["dep:reth-eth-wire"]
```

Comment:
for this we don't need an extra feature because this is already imported via reth-network so we can just enable this if network feature is enabled